docs/csharp/language-reference/operators/lambda-expressions.md

```diff
@@ -216,10 +229,8 @@ The following rules apply to variable scope in lambda expressions:
 
 For more information, see the [Anonymous function expressions](~/_csharplang/spec/expressions.md#anonymous-function-expressions) section of the [C# language specification](~/_csharplang/spec/introduction.md).
 
-## Featured book chapter
+For more information about lambda discard parameters, see the [feature proposal note](~/_csharplang/proposals/csharp-9.0/lambda-discard-parameters.md)
 
-[Delegates, Events, and Lambda Expressions](/previous-versions/visualstudio/visual-studio-2008/ff518994(v=orm.10)) in [C# 3.0 Cookbook, Third Edition: More than 250 solutions for C# 3.0 programmers](/previous-versions/visualstudio/visual-studio-2008/ff518995(v=orm.10))  
```

Comment:
This article has been updated considerably during the last years; I guess, it's very far from that original chapter now (which is for C# 3)

Reply:
Yes, that's true.

Editorial tidbit: When I last updated Effective C# and More Effective C# (for versions 6 & 7, respectively), one of my reviewers made the point of commenting whenever I missed a reference to a "new" feature from C# 3 (the previous updates.)